official/ubuntu_18_04/docker/ros2/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# [BASH IGNORE]
+
+FROM docker.is.localnet:5000/amd/18.04:core
+MAINTAINER Vincent Berenz <vincentberenz@tuebingen.mpg.de>
+
+# [/BASH IGNORE]
+
+####################################################
+#             ROS INSTALLATION
+####################################################
+
+# Setup Local.
+RUN locale-gen en_US en_US.UTF-8
+RUN update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+RUN export LANG=en_US.UTF-8
+
+# Setup Sources.
+RUN apt update && sudo apt install curl gnupg2 lsb-release
```

Comment:
For the `apt install`, should there be a `apt install -y` flag `-y` to make sure it installs without asking the user for input? Same below.

Reply:
Fair point I will change that